src/main/java/com/imgix/Validate.java

```diff
@@ -0,0 +1,53 @@
+package com.imgix;
+
+public class Validate {
+    /**
+     * Validates the width range specified by `begin` and `end`.
+     *
+     * A width range is valid if `begin` is less than or equal to
+     * `end` _and_ if `begin` is greater than zero.
+     *
+     * The maximum value for `end` is generally 8192. However, this
+     * is only the general case. In the future, values larger than
+     * 8192 will become valid.
```

Comment:
Nit: I don't think we should be publicly promising this.
```suggestion
     * The maximum value for `end` is codified as 8192, in
     * accordance with the imgix rendering spec. However, we
     * are not establishing a hard validation for it herer, in 
     * the case that it changes at some point in the future.
```

Reply:
I agree on the first part; have questions about this second part, but will remove this section nonetheless.